releasenotes/notes/0.7/subexperiment-gen-speedup-41a4e8679353d1d9.yaml

```diff
@@ -1,4 +1,4 @@
 ---
-fixes:
+other:
   - |
     The :func:`.generate_cutting_experiments` function has been optimized for faster execution.
```

Comment:
@caleb-johnson I know I claimed previously that this should be `fixes` instead of `upgrade`, but after previewing the release notes I actually think it makes even more sense to be `other`, i.e., under "other notes" rather than "bug fixes".

![0 7 1-notes](https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/assets/91987/db484914-b8d3-48f0-8a23-85f7f5d3dbbf)